src/main/java/com/LP2/EventScheduler/service/event/EventServiceImpl.java

```diff
@@ -153,6 +154,22 @@ public MessageResponse joinEvent(UUID eventId, JoinEventDTO joinData, User user)
 
         return new MessageResponse("Participation added");
     }
+    @Override
+    public MessageResponse removeEvent(UUID eventId, DeleteEventDTO deleteData, User user){
+        Event event = this.eventRepository
+                .findById(eventId)
+                .orElseThrow(EventNotFoundException::new);
+        if (event.getStatus() != EventStatus.CANCELLED) {
+            throw new IsNotOwnerException("Event must have status CANCELLED to be deleted");
```

Comment:
En este caso seria mejor utilizar la excepción `UnexpectedResourceValueException`
```suggestion
            throw new UnexpectedResourceValueException("Event must have status CANCELLED to be deleted");
```